tools/wpt/virtualenv.py

```diff
@@ -81,7 +81,11 @@ def lib_path(self):
             if IS_WIN:
                 site_packages = os.path.join(base, "Lib", "site-packages")
             else:
-                site_packages = os.path.join(base, "lib", f"python{sys.version[:3]}", "site-packages")
+                version = sys.version[:3]
```

Comment:
Maybe we could just write:
```
version = f"{sys.version_info.major}.{sys.version_info.minor}"
```
or something similar that doesn't depend on string parsing (or at least, do something like `".".join(sys.version.split(".", 2)[:2])` which just assumes we want the first two dot-separated components).